<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoom</class>
 <widget class="QWidget" name="NewRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>591</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lableImage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>创建房间信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>房间类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>房间价格(/晚)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>房间数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>房间折扣</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePrice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDiscount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>选填</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDiscription">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>选填</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>房间描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveImage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>钟点房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大床房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双人房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>奢华房</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>总统套房</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>图片名称：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
